### ISSUE TITLE
How to increase loss scalar frequency while using Tensorboard callback?

### ISSUE THREAD

**Issue by BrikerMan** [fchollet/keras#7547]
Thank you for the amazing work.

I am using Tensorboard to visualize my training, and it works just fine.

But when I need to change the frequency of loss scalers, I can't find api to do this. I want to more scalers log, maybe log end of every batch. Now I can only get one log during an epoch. Thanks a lot.

```
tensorboard_callback = TensorBoard(log_dir = log_dir,
    histogram_freq = 1,
    write_graph = True,
    write_images = False,
    embeddings_freq = embeddings_freq,
    embeddings_layer_names = None,
    embeddings_metadata = "w2v_metadata.tsv")
```

- [x ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by BrikerMan** [comment]
duplicated #6692
